docs/testplan/pac/Port_Access_Control.md

```diff
@@ -0,0 +1,566 @@
+| NAS        | Network Access Switch                    |
+| DUT        | Device Under Test                        |
+| RADIUS     | Remote Authentication Dial In User service |
+| FDB        | Forwarding Database                      |
+| Supplicant | A client that attempts to access services offered by the Authenticator |
+| PAE        | Port Access Entity                       |
+
+## Introduction
+
+### Objective 
+
+The main objective of this document is to cover the test cases that will be executed for Port authentication methods 802.1x and MAB. Topologies and test cases for testing the feature will be discussed as part of this document.
+
+### Scope
+
+- PAC authentication of hosts on access port 
+
+### Out of scope
+
+- Authentication on Trunk port not supported
+
+## Feature Overview
+
+Port Access Control (PAC) feature provides validation of client and user credentials to prevent unauthorized access to a specific switch port.
+
+Local Area Networks (LANs) are often deployed in environments that permit unauthorized devices to be physically attached to the LAN infrastructure, or permit unauthorized users to attempt to access the LAN through equipment already attached. In such environments, it may be desirable to restrict access to the services offered by the LAN to those users and devices that are permitted to use those services. Port access control makes use of the physical characteristics of LAN infrastructures in order to provide a means of authenticating and authorizing devices attached to a LAN port that has point-to-point connection characteristics and of preventing access to that port in cases in which the authentication and authorization process fails. In this context, a port is a single point of attachment to the LAN, such as Ports of MAC bridges and associations between stations or access points in IEEE 802.11 Wireless LANs.
+
+802.1x:
+
+IEEE 802.1X-2004 is an IEEE Standard for Port Access Control (PAC) that provides an authentication mechanism to devices wishing to attach to a LAN. The standard defines Extensible Authentication Protocol Over LAN (EAPoL). The 802.1X standard describes an architectural framework within which authentication and consequent actions take place. It also establishes the requirements for a protocol between the authenticator and the supplicant, as well as between the authenticator and the authentication server.
+
+MAC Authentication Bypass(MAB):
+
+Simple devices like camera or printers which do not support 802.1x authentication can make use of MAB feature where the device gets authenticated based on the device MAC address.
+
+## 1 Test Focus Areas
+
+### 1.1 CLI Testing
+
+	- Verify port authentication can be enabled only on physical interfaces and gets denied on VLAN, Portchannel, PO member ports and sub interfaces.
+	- Verify configured CLI fields are updated properly in respective show commands
+	- Verify PAC configurations/show outputs via REST URI
```

Comment:
Pls clarify the REST URI test is on SONIC-YANG

Reply:
Addressed the comments